package.json

```diff
@@ -37,6 +37,7 @@
     "eslint": "latest",
     "esm": "latest",
     "mkdirp": "latest",
+    "pirates": "latest",
```

Comment:
`lib/` is not bundled and adding to `dependencies`, makes jiti no longer zero dep. Maybe we can expose `register` from src/jiti (jiti instance)?

Reply:
Great idea. Will implement